x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/EsqlExpressionTranslators.java

```diff
@@ -401,4 +404,58 @@ static Query translate(
             }
         }
     }
+
+    public static class InComparisons extends ExpressionTranslator<In> {
```

Comment:
I think we're moving these into their own top level file more often these days.

Reply:
> I think we're moving these into their own top level file more often these days.

I just double checked, a lot of `LogicalPlanOptimizer` rules are in their own top level files, under optimizer/rules, the ExpressionTranslators are not in their own files yet. They are in either `EsqlExpressionTranslators` or `ExpressionTranslators` in esql-core. Perhaps keep it here for now?

Reply:
Either is fine by me.